wob.c

```diff
@@ -256,7 +256,7 @@ main(int argc, char **argv)
 		return EXIT_FAILURE;
 	}
 
-	uint32_t * argb = wob_create_argb_buffer(&app), i;
+	uint32_t * argb = wob_create_argb_buffer(&app), i,k;
```

Comment:
Add space after comma.

```suggestion
	uint32_t * argb = wob_create_argb_buffer(&app), i, k;
```

Reply:
Fixed.